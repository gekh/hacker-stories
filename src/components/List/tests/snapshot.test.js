import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import { List, Item } from '..';
import { storyOne, stories } from '../../../fixtures/stories.fixture';


describe('List', () => {
  it('renders a snapshot', () => {
    const { container } = render(<List list={stories} onRemoveItem={jest.fn()} />);
    expect(container.firstChild).toMatchSnapshot();
  });
});

describe('Item', () => {
  it('renders a snapshot', () => {
    const { container } = render(<Item item={storyOne} onRemoveItem={jest.fn()} />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
