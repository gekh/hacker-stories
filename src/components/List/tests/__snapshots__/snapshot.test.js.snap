// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Item renders a snapshot 1`] = `
<li
  class="sc-hLBbgP eDqXDW"
>
  <span
    class="sc-eDvSVe feTYso"
    width="40%"
  >
    <a
      href="https://reactjs.org/"
    >
      React
    </a>
  </span>
  <span
    class="sc-eDvSVe feSJiH"
    width="30%"
  >
    Jordan Walke
  </span>
  <span
    class="sc-eDvSVe feSkhN"
    width="10%"
  >
    3
  </span>
  <span
    class="sc-eDvSVe feSkhN"
    width="10%"
  >
    4
  </span>
  <span
    class="sc-eDvSVe feSkhN"
    width="10%"
  >
    <button
      class="sc-bcXHqe sc-gswNZR cctTZy hmpeyq"
      type="button"
    >
      Dismiss
    </button>
  </span>
</li>
`;

exports[`List renders a snapshot 1`] = `
<ul>
  <li
    class="sc-hLBbgP eDqXDW"
  >
    <span
      class="sc-eDvSVe feTYso"
      width="40%"
    >
      <a
        href="https://reactjs.org/"
      >
        React
      </a>
    </span>
    <span
      class="sc-eDvSVe feSJiH"
      width="30%"
    >
      Jordan Walke
    </span>
    <span
      class="sc-eDvSVe feSkhN"
      width="10%"
    >
      3
    </span>
    <span
      class="sc-eDvSVe feSkhN"
      width="10%"
    >
      4
    </span>
    <span
      class="sc-eDvSVe feSkhN"
      width="10%"
    >
      <button
        class="sc-bcXHqe sc-gswNZR cctTZy hmpeyq"
        type="button"
      >
        Dismiss
      </button>
    </span>
  </li>
  <li
    class="sc-hLBbgP eDqXDW"
  >
    <span
      class="sc-eDvSVe feTYso"
      width="40%"
    >
      <a
        href="https://redux.js.org/"
      >
        Redux
      </a>
    </span>
    <span
      class="sc-eDvSVe feSJiH"
      width="30%"
    >
      Dan Abramov, Andrew Clark
    </span>
    <span
      class="sc-eDvSVe feSkhN"
      width="10%"
    >
      2
    </span>
    <span
      class="sc-eDvSVe feSkhN"
      width="10%"
    >
      5
    </span>
    <span
      class="sc-eDvSVe feSkhN"
      width="10%"
    >
      <button
        class="sc-bcXHqe sc-gswNZR cctTZy hmpeyq"
        type="button"
      >
        Dismiss
      </button>
    </span>
  </li>
</ul>
`;
